File: javascript/challenges/arrayShift/array-shift.js
function insertShiftArray(arr, val){
  let midpoint = Math.round(arr.length / 2);
  let newArray = [];
  for (let i=0; i<midpoint; i++){
    newArray.push(arr[i])
  }
  newArray.push(val);
  for (let i=midpoint; i<arr.length; i++){
    newArray.push(arr[i])}
  return newArray;
}

File: python/array_shift/array_shift.py
# Write a function called insertShiftArray which takes in an array and the value to be added. Without utilizing any of the built-in methods available to your language, return an array with the new value added at the middle index.

def array_shift(list, val):
  midpoint = round(len(list)/2)
  list.insert(midpoint, val)
  return list
